lib/features/context-pad/ContextPadProvider.js

```diff
@@ -420,7 +420,8 @@ ContextPadProvider.prototype.getContextPadEntries = function(element) {
         'append.gateway': appendAction(
           'bpmn:ExclusiveGateway',
           'bpmn-icon-gateway-none',
-          translate('Append gateway')
+          translate('Append gateway'),
+          { isMarkerVisible: true }
```

Comment:
This change won't be necessary if we implement what I suggested in https://github.com/bpmn-io/bpmn-js/pull/2071/files/10ca198b1d479805812eaa473d31b15a607358aa#r1447241633